glide.yaml

```diff
@@ -44,3 +44,8 @@ import:
   - quantile
 - package: github.com/uber-go/atomic
 - package: github.com/prashantv/protectmem
+- package: github.com/opentracing/opentracing-go
+- package: github.com/uber/jaeger-client-go
+- package: github.com/opentracing/basictracer-go
+- package: github.com/gogo/protobuf
```

Comment:
This shouldn't be in the glide.yaml, I don't think tchannel-go is directly using protobuf anywhere right?